web-test-runner.config.mjs

```diff
@@ -17,6 +18,10 @@ function customReporter() {
   };
 }
 export default {
+  playwright: true,
+  browsers: [
+    playwrightLauncher({ product: 'chromium', launchOptions: { headless: true } }),
```

Comment:
Added benefit, in the future we could run the unit tests on more products. With the big caveat that currently most of the unit tests fail on both firefox and webkit.
```
    playwrightLauncher({ product: 'chromium' }),
    playwrightLauncher({ product: 'firefox' }),
    playwrightLauncher({ product: 'webkit' }),
```